contracts/SafeProtocolRegistry.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity ^0.8.18;
+import "./interfaces/ISafeProtocolRegistry.sol";
+import "@openzeppelin/contracts/access/Ownable2Step.sol";
+
+contract SafeProtocolRegistry is ISafeProtocolRegistry, Ownable2Step {
+    mapping(address => ComponentInfo) listedComponents;
+
+    struct ComponentInfo {
+        uint256 listedAt;
```

Comment:
These are timestamps, therefore `uint256` is unnecessary. I would recommend a `uint64`